app/controllers/miq_ae_customization_controller.rb

```diff
@@ -67,6 +67,12 @@ def upload_import_file
 
   def import_service_dialogs
     if params[:commit] == _('Commit')
+      if params[:dialogs_to_import].blank?
+        add_flash(_("At least one Service Dialog must be selected."), :error)
+        render_flash_and_stop_sparkle
```

Comment:
`render_flash_and_stop_sparkle(_("At least one Service Dialog must be selected."), :error)`

will call the `add_flash` for you if I am reading it correctly